README.md

```diff
@@ -45,10 +45,30 @@ Mixins within `o-brand` help configure components to support brands. There is no
 
 The following mixins and functions help brand a component.
 
+- [oBrandGetCurrentBrand](#obrandgetcurrentbrand) - Set a default brand if necessary
 - [oBrandDefine](#obranddefine) - Define brand configuration (variables & supported variants).
 - [oBrandGet](#obrandget) - Retrieve brand variables.
 - [oBrandSupportsVariant](#obrandsupportsvariant) - Check if the brand supports a variant.
 
+### oBrandGetCurrentBrand
+
+This function will return the brand defined at a product level, or can be used to set a brand to be used within a component.
```

Comment:
This read a little weird to me:
>This function will return the brand defined at a product level, or can be used to set a brand to be used within a component.

Something like this?
>This function will return the brand defined at a product level. It can be used to define a brand within a component conditionally.

Reply:
Let's also update the `oBrandDefine` example to be wrapped in a `oBrandGetCurrentBrand` conditional.